.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,8 @@
+Related to #<ID>
+
+Acceptance criteria 
+=================== 
+- [ ] Functionality implemented in agreement with description of the ticket
+- [ ] Tests are updated and pass 
```

Comment:
for this i would keep it to "Tests are updated". The "and pass" part is checked by the pipeline and not by the reviewer